src/domain/session/rightpanel/RoomInfoViewModel.js

```diff
@@ -0,0 +1,50 @@
+import { ViewModel } from "../../ViewModel.js";
+import { avatarInitials, getIdentifierColorNumber, getAvatarHttpUrl } from "../../avatar.js";
+
+export class RoomInfoViewModel extends ViewModel {
+    constructor(options) {
+        super(options);
+        this._room = options.room;
```

Comment:
To receive updates on any properties of the room, you'll need to listen to its `change` event and call `emitChange`. Currently, if the room name, avatar or join count, ... would change, the right panel would not update. RoomViewModel does exactly the same, so serves as a good example.